src/account/AccountSign.ts

```diff
@@ -7,7 +7,7 @@ export interface IAccountSignRequest {
 }
 
 /**
- * This interface represents the response object containing the signed message data, which is sent by the content script 
+ * This interface represents the response object containing the signed message data, which is sent by the content script
```

Comment:
Those are linter fix. they should be in another PR. Plus we should have a CI checking all that. Can you create an issue for the CI to check the linter ?

Reply:
done